src/updatedownloader.cpp

```diff
@@ -222,7 +151,7 @@ void UpdateDownloader::CleanLeftovers()
     // malicious users from forcing us into deleting arbitrary directories:
     try
     {
-        if (tmpdir.find(GetUniqueTempDirectoryPrefix()) != 0)
+        if (tmpdir.find(winsparkle::GetUniqueTempDirectoryPrefix()) != 0)
```

Comment:
Is name space designation `winsparkle::` really needed here when already inside the namespace?